### ISSUE TITLE
Entmax alpha implementation

### ISSUE THREAD

**Issue by rwbfd** [tensorflow/addons#2312]
I would like to implement the entmax-alpha feature as is described in https://arxiv.org/abs/1905.05702 in the addons. For alpha=1, it is softmax, and for alpha=2, it is sparsemax.

This feature is a key to realize sparse seq-to-seq model. It will not change the current API.

I have almost finished implementing the feature. Therefore, I would like to know if this has any chance of being merged before continuing.

**Comment by bhack** [comment]
I think we need to evaluate this for https://github.com/tensorflow/addons/issues/2194
If It pass the evaluation we will put this in `ecosystem-review`

**Comment by rwbfd** [comment]
@bhack Thanks for the timely reply.

**Comment by bhack** [comment]
@seanpmorgan @WindQAQ this doesn't have enough citations but it will not change the API. What do you think?

**Comment by seanpmorgan** [comment]
If API is unchanged and it's a small PR then I'm good with it

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
